src/WP_CLI/CommandWithTerms.php

```diff
@@ -121,11 +121,12 @@ public function list_( $args, $assoc_args ) {
 	 * : The name of the term's taxonomy.
 	 *
 	 * [<term>...]
-	 * : The name of the term or terms to be removed from the object.
+	 * : The slug of the term or terms (space-separated) to be removed from the object.
```

Comment:
```suggestion
	 * : The slug of the term or terms to be removed from the object.
```

"(space-separated)" isn't necessary here because it's the same behavior across all commands.

Reply:
Hi @danielbachhuber 

I've added `space-separated` through all commands so the user can have this information without having to visit another page. Do you think this is not necessary because of the examples added in the `wp post term` page?

Reply:
> Do you think this is not necessary because of the examples added in the `wp post term` page?

Yes, and that it's standard behavior across all WP-CLI commands.

Reply:
Got it. Removed.